thoth/common/logging.py

```diff
@@ -148,6 +148,19 @@ def _get_sentry_integrations() -> List[object]:
 
     return integrations
 
+def before_send_handler(event, hint):
+    """ Ignores the exceptions passed in as as the environment variable in a comma separated manner.
+    """
+    ignored_exceptions = os.environ("THOTH_SENTRY_IGNORE_EXCEPTION")
+    print(ignored_exceptions)
+    if ignored_exceptions:
+        if 'exc_info' in hint:
+            exc_type, exc_value, tb = hint['exc_info']
+        for exception in ignored_exceptions.split(','):
```

Comment:
This block should be executed only for if exc_info is present - we can send also errors into sentry using `_LOGGER.error()` which will not hold any exception.

Reply:
Thanks! :smile: